content/wasm-functions/using-cron-trigger.md

```diff
@@ -32,16 +34,158 @@ Use Cases for Cron Jobs:
 * Log file rotation
 * Scheduled notifications
 
-# Spin and Fermyon Wasm Functions
+# Create Spin application 
 
-Spin is a lightweight framework for building WebAssembly microservices and applications. It allows developers to define cron jobs that execute functions at scheduled intervals. To simplify development, Spin provides templates for various use cases, including scheduled tasks. 
+You must have a running Spin application on Fermyon Wasm Functions to apply the cron trigger to. We'll create a new Spin application for scheduling tasks using cron jobs using the `http-js` template:
 
-## Create Spin application 
+<!-- @selectiveCpy -->
 
-You must have a running Spin application on Fermyon Wasm Functions to apply the cron trigger to. We'll create a new Spin application for scheduling tasks using cron jobs:
+```console
+$ spin new -t http-js --accept-defaults hello-cron-job
 
+$ cd hello-cron-job
 
+$ npm install
+```
 
-## Creating a Cron Job for your Spin application
+We'll go ahead add a specific path for our cron trigger in later steps:
 
-## Managing Cron Jobs
+```toml
+[[trigger.http]]
+route = "/hello"
+component = "hello-cron-job"
+```
+
+Let's modify the response body so we can validate that our application is running on a specific interval. To do so, navigate to `src/index.js` and add the following snippet of code:
+
+<!-- @selectiveCpy -->
+
+```JavaScript
+// For AutoRouter documentation refer to https://itty.dev/itty-router/routers/autorouter
+import { AutoRouter } from 'itty-router';
+
+let router = AutoRouter();
+
+// Route ordering matters, the first route that matches will be used
+// Any route that does not return will be treated as a middleware
+// Any unmatched route will return a 404
+router
+    .get("/hello", () => {
+    // Grab datetime
+    const now = new Date().toISOString();
+    // Return response with time of invocation
+    return new Response(`Hello from a cron component at ${now}`, { status: 200 });
+});
+
+
+addEventListener('fetch', (event) => {
+    event.respondWith(router.fetch(event.request));
+});
+```
+
+To test locally, run the following command:
+
+<!-- @selectiveCpy -->
+
+```console
+spin build --up
+```
+
+If it is working as expected, you should see a message returned along the lines of (but with an accurate datetime):
```

Comment:
To me this reads like I should see the message from `spin up`, but you actually mean when I curl it.